src/cmd/tags_delete.go

```diff
@@ -112,6 +117,7 @@ var (
 func init() {
 
 	tagsDeleteCmd.Flags().BoolVar(&dryRunFl, "dry-run", false, "show what would be deleted without actually deleting any tags")
+	tagsDeleteCmd.Flags().BoolVar(&yesFl, "yes", false, "suppress command line prompts and confirm command execution")
```

Comment:
This need to include the shorthand flag `-y`. Also, the description should be improved, it's too generic.

Reply:
1. Could you share what you would like the description to be?

2. I'm looking into adding an alias for `--yes` as `--y`, will need some time to research. Could you point me in the right direction?

Reply:
1. "automatic yes to deletion prompt, useful for scripting"
2. [link](https://github.com/spf13/cobra/blob/master/README.md)